File: 3_public_encapsulation.cpp
#include<bits/stdc++.h>
using namespace std;
class Person
{
    public:
        string name;
        int age;

        Person(string name, int age)
        {
            this->name = name;
            this->age = age;
        }

        void myData()
        {
            cout<<name<<" "<<age<<endl;
        }
};
int main()
{
    Person rahim("Rahim", 25);
    cout<<rahim.name<<" "<<rahim.age<<endl;
    rahim.myData();
    return 0;
}

File: 3_public_encapsulation.py
# public encapsulation
class Person:
    def __init__(self, name, age) -> None:
        self.name = name
        self.age = age

    def myData(self):
        return f"{self.name} {self.age}"
    
rahim = Person('rahim', 25)
print(rahim.name, rahim.age)
print(rahim.myData())